framework/include/controls/RealFunctionControl.h

```diff
@@ -62,7 +62,7 @@ class RealFunctionControl : public Control
   Function & _function;
 
   /// Vector of parameters to change
-  std::vector<Real *> _parameters;
+  ControllableParameter<Real> _parameters;
```

Comment:
I think getting them is important, what if what one does with a parameter is a function of the parameter itself. It just seems weird that we can change the parameter but we can't inspect its value.